{{cookiecutter.repository_slug}}/AUTHORS.md

```diff
@@ -0,0 +1,9 @@
+# Authors
+
+## Core Contributor
+
+{% if cookiecutter.remote_provider != "none" %}[{{ cookiecutter.author_name }}](https://{{ cookiecutter.remote_provider }}/{{ cookiecutter.remote_username }}) &lt;{{ cookiecutter.author_email }}&gt;{% else %}{{ cookiecutter.author_name }} &lt;{{ cookiecutter.author_email }}&gt;{% endif %}
+
+## Contributors
+
+None yet. [Why not be the first?](CONTRIBUTING.md)
```

Comment:
I'm not a fan of AUTHORS files, as I find them redundant, and they add extra work for contributors, or introduce inconsistencies in credits when people forget to add themselves in it.

When people contribute to the repo with a PR, then they are listed in the git history (and in the Contributors tab on GitHub). It should be enough. I know all contributions are not git commits, but then it's a very subjective choice who you include, why, and where. I would prefer to remove the AUTHORS file from the generated contents (the one at the root of this repo can stay, my fault for putting it there in the first place :stuck_out_tongue:).

Reply:
Now I think we should drop the AUTHORS too ^^

Reply:
Good, so we drop the templated one, and keep the one at the root of the project :slightly_smiling_face: